File: MyVideoApp/app/src/main/java/com/example/abhishekyadav/myvideoapp/MainActivity.java
package com.example.abhishekyadav.myvideoapp;

import android.content.Intent;
import android.net.Uri;
import android.provider.MediaStore;
import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.widget.VideoView;


public class MainActivity extends AppCompatActivity {

    static final int REQUEST_VIDEO_CAPTURE = 1;
    VideoView result;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);


        result = (VideoView) findViewById(R.id.videoView);
    }

    public void dispatchTakeVideoIntent() {
        Intent takeVideoIntent = new Intent(MediaStore.ACTION_VIDEO_CAPTURE);
        if (takeVideoIntent.resolveActivity(getPackageManager()) != null) {
            startActivityForResult(takeVideoIntent, REQUEST_VIDEO_CAPTURE);
        }
    }

    @Override
    protected void onActivityResult(int requestCode, int resultCode, Intent intent) {
        if (requestCode == REQUEST_VIDEO_CAPTURE && resultCode == RESULT_OK) {
            Uri videoUri = intent.getData();
            result.setVideoURI(videoUri);
        }
    }
}

File: SumApp/app/src/main/java/com/example/abhishekyadav/sumapp/MainActivity.kt
package com.example.abhishekyadav.sumapp

import android.support.v7.app.AppCompatActivity
import android.os.Bundle
import android.view.View
import android.widget.Button
import kotlinx.android.synthetic.*
import kotlinx.android.synthetic.main.activity_main.*
import kotlinx.android.synthetic.main.activity_main.view.*


class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
    }

    public fun onButtonClick(view:View)
    {
        val e1 = editText
        val e2 = editText2
       val num1 = Integer.parseInt(e1.text.toString())
       //val num1 = e1.text.toString()
        val num2 = Integer.parseInt(e2.text.toString())
        val sum = num1 + num2
        //val sum = 4
        textView.setText(Integer.toString(sum))

    }
}
